test/drenv/addons/rook/rbd_mirror/test.py

```
@@ -96,30 +122,36 @@ def test_volume_replication(primary, secondary):
     kubectl.get(
-        f"volumereplication/{VR_NAME}",
+        f"{kind}/{name}",
         "--output=yaml",
         f"--namespace={NAMESPACE}",
         context=primary,
     )
 
+
+def _rbd_mirror_status(primary, image):
     print(f"rbd mirror image status in cluster '{primary}'")
     image_status = _rbd_mirror_image_status(primary, rbd_image)
```

Ruff (0.16.3)

[error] Undefined name `rbd_image`

(F821)